### PR TITLE
[ DONE ][psr_logger] integration as independet service

### PR DESCRIPTION
**Pull request by mgrinko**
@makasim ready to be merged

### CODE REVIEW

File: Resources/config/services.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
         <parameter key="fp_badaboom.exception_catcher.class">Fp\BadaBoomBundle\ExceptionCatcher\ExceptionCatcher</parameter>
         <parameter key="fp_badaboom.exception_listener.class">Fp\BadaBoomBundle\EventListener\ExceptionListener</parameter>
         <parameter key="fp_badaboom.chain_node_manager.class">Fp\BadaBoomBundle\ChainNode\ChainNodeManager</parameter>
+        <parameter key="fp_badaboom.psr_logger.class">BadaBoom\Bridge\Psr\Logger</parameter>
```
Comment (makasim) [inline Resources/config/services.xml]: I would name it without `psr_` prefix. Like this `fp_badaboom.logger.class`

Comment (makasim) [inline Resources/config/services.xml]: Let me know when you done with this change. I am going to merge it.

Comment (mgrinko) [inline Resources/config/services.xml]: Ready

File: Tests/Integration/ContainerTest.php [hunk 2]
```diff
@@ -75,6 +75,16 @@ public function shouldGetSerializerWhichSupportsEncodingToLineFormat()
         $this->assertTrue($serializer->supportsEncoding('line'));
     }
 
+    /**
+     * @test
+     */
+    public function shouldGetPsrLogger()
+    {
+        $logger = self::$container->get('fp_badaboom.psr_logger');
```
Comment (makasim) [inline Tests/Integration/ContainerTest.php]: what about it?

**Pull request merged**